package.json

```diff
@@ -68,6 +68,7 @@
     "@typescript-eslint/parser": "^8.7.0",
     "@vitest/coverage-istanbul": "^3.0.8",
     "eslint": "^9.17.0",
+    "hono": "*",
```

Comment:
Setting a single version in the root is enough to satisfy the peer dependencies of each workspace. It also ensures that there's only a single version installed at any one time.